/**:
  ros__parameters:
    overhang: [0.2, 0.4, 0.1, 0.1] # front, rear, left, right
    height: 1.0
    wheel_tread: 1.5
    wheel_base: 2.8
    mass: 1500.0
    max_speed: 20.0 # 72.0 [km/h]
    max_accel: 5.0
    max_brake: 5.0
    max_steer: 0.785398 # 45 [deg]
